tests/unit/components/textInput.test.js

```diff
@@ -107,8 +97,22 @@ describe('textInput', () => {
     expect(container.querySelector(`[data-testid="${textInputName}"]`).value).toBe(value);
   })
 
+  test('Displays autoComplete', () => {
```

Comment:
This is testing an attribute, not that something is display - please update the test title
```suggestion
  test('Text input has correct autoComplete attribute', () => {
```